src/time.cr

```diff
@@ -605,10 +605,19 @@ struct Time
     {% end %}
 
     unless offset
-      # current TZ may have DST, either in past, present or future
-      ret = LibC.localtime_r(pointerof(second), out tm)
-      raise Errno.new("localtime_r") if ret.null?
-      offset = tm.tm_gmtoff.to_i64
+      {% if !flag?(:windows) %}
+        # current TZ may have DST, either in past, present or future
+        ret = LibC.localtime_r(pointerof(second), out tm)
+        raise Errno.new("localtime_r") if ret.null?
+        offset = tm.tm_gmtoff.to_i64
+      {% else %}
+        # TODO handle daylight?
+        # https://msdn.microsoft.com/en-us/library/windows/desktop/ms724421(v=vs.85).aspx
+        ret = LibWindows.get_time_zone_information(out tz)
+        raise WinError.new("GetTimeZoneInformation") if ret == -1
+        # UTC = local time + bias
+        return tz.bias.to_i64 * - Span::TicksPerMinute
+      {% end %}
```

Comment:
I'm fine with the pedantic, yet sometimes I would prefer the oposite. So the most used branch shows before. I'll change it later.

Regarding the DST, I think it is not needed (hence the `?`). I change clock to a timezone with daylight back and forth. And clock current time and (real) current time showed fine. 

I hope the specs will point out issues when the spec framework will be able to be used in windows.

But if you find a case that is not handled let me know.